langs/sed/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.15 as builder
+
+RUN mkdir /empty
+
+RUN apk add --no-cache build-base curl
+
+RUN curl https://ftp.gnu.org/gnu/sed/sed-4.8.tar.gz | tar xz
+
+RUN cd sed-4.8                    \
+ && ./configure LDFLAGS="-static" \
+ && make                          \
+ && strip sed/sed
+
+FROM scratch
```

Comment:
There's a new base package called `codegolf/lang-base` which will let you drop the `/empty` lines from here and the mkdir from above.

Reply:
Everything should be addressed now